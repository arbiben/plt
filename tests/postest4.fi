int a?
char c?
bool b?
arr ar?

struct class {
int size?
arr people?
}

bool correct(arr people, int file1, arr sentence) {
open(file1)? $ random comment here $
close(file1)?
print(sentence, file1, people)?
ar = ['a', 'b', '?', 'b']?
c = ar[3]?
if (c <= 3) {b = T?} else {b = F?}
}

$
checking comments work
Yeah yep
$
