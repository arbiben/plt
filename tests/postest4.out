struct class {
int size?
arr people?
}

int a?
char c?
bool b?
arr ar?

bool correct(people, file1, sentence)
{
open(file1)?
close(file1)?
print(sentence, file1, people)?
ar = ar2 = ['a', 'b', '?', 'b']?
c = ar[3]?
if (c <= 3)
{
b = T?
}
else
{
b = F?
}
}

